packages/components/sidebar/src/sidebar-item.tsx

```diff
@@ -74,12 +80,29 @@ export default function SidebarItem({
         </div>
       )}
       {expanded ? (
-        <div className={sidebarItemTitleWrapperClsx()}>
-          <span className={sidebarItemTitleClsx({ disabled })}>{title}</span>
-          {Boolean(badge) && (
-            <span className={sidebarItemTitleBadgeClsx()}>{badge}</span>
-          )}
-        </div>
+        <>
+          <div className={sidebarItemInfoWrapperClsx()}>
+            <div className={sidebarItemTitleWrapperClsx()}>
+              <Typography
+                level="p5"
```

Comment:
Since this is for title (as for the variable name), a heading tag would be more appropriate?